### PR TITLE
Remove unused variables from WhiskConfig and add the ability to Confi…

### PR DESCRIPTION
**Pull request by ChrBi**
…g to ask for environment-variables.

### CODE REVIEW

File: common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala [hunk 1]
```diff
@@ -17,20 +17,19 @@
...
-import whisk.core.WhiskConfig.consulServices
 import whisk.core.WhiskConfig.invokerHosts
 
 object ConsulServiceHealth extends App {
 
-    def checkHealth(consulServer: String) = {
-        val requiredProperties = consulServices ++ invokerHosts
+    def checkHealth(consulServer: String, consulServices: String) = {
+        val requiredProperties = invokerHosts
         val config = new WhiskConfig(requiredProperties)
```
Comment (markusthoemmes) [inline common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala]: now unused?

**Comment by rabbah** [comment]
PG?

**Comment by ChrBi** [comment]
PG 122 was green.
I'll rebase and start a newer one.

File: common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala [hunk 2]
```diff
@@ -15,50 +15,40 @@
...
+        val passing = consul.getAllPassing()
+        val critical = consul.getAllCritical()
+        val warning = consul.getAllWarning()
+        val nonpassing = servicesToCheck -- passing
+        val notfound = nonpassing -- critical -- warning
+        // check that all known services are passing
+        if (nonpassing.isEmpty) {
+            println("All passing")
+        }
+        if (!critical.isEmpty) {
```
Comment (rabbah) [inline common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala]: why not `if (critical.nonEmpty) ...`?

File: common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala [hunk 3]
```diff
@@ -15,50 +15,40 @@
...
-            }
-            if (!notfound.isEmpty) {
-                print("Unknown: ")
-                notfound.foreach(service => print(service + " "))
-                println
-            }
-        } else {
-            println("No valid config")
+        // retrieve set of services to check and invoker0 to the list which exists in all deployments
+        // TODO: add invokers specific to a deployment
```
Comment (rabbah) [inline common/scala/src/main/scala/whisk/common/ConsulServiceHealth.scala]: can we get rid of this todo? get the invokers from consul and include in services to check?

**Comment by rabbah** [comment]
Made the suggested change.

Merged as https://github.com/openwhisk/openwhisk/commit/9882e3ad61f9d614ecd27e661dbba41e386842ea.

**Pull request closed without merging**